models/adapters/bing_ad_performance_adapter.sql

```diff
@@ -8,19 +8,24 @@ bing_keyword_performance_agg as (
 
     select 
         campaign_date,
+        keyword_id as criteria_id,
+        ad_group_id,
+        ad_group_name,
         url_host,
         url_path,
         utm_source,
         utm_medium,
         utm_campaign,
         utm_content,
         utm_term,
+        campaign_id,
+        campaign_name,
         'bing ads' as platform,
         sum(clicks) as clicks,
         sum(impressions) as impressions,
         sum(spend) as spend
     from bing_keyword_performance
-    group by 1, 2, 3, 4, 5, 6, 7, 8, 9
+    group by 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14
```

Comment:
can you switch this to `dbt_utils.group_by()`?

Reply:
done